### PR TITLE
Update docs for 2.0.0 release

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@guozhangwang I have regenerated the docs for 2.0.0, can you review please? Thank you!

File: 20/javadoc/org/apache/kafka/server/quota/ClientQuotaCallback.html [hunk 1]
```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (guozhangwang) [inline 20/javadoc/org/apache/kafka/server/quota/ClientQuotaCallback.html:1]: The added javadoc seems right to me as https://issues.apache.org/jira/browse/KAFKA-6576 is supposed to be included in 2.0. cc @rajinisivaram

File: 20/streams/developer-guide/processor-api.html [hunk 2]
```diff
@@ -354,7 +354,7 @@ <h2>
         </div>
         <div class="section" id="accessing-processor-context">
             <h2><a class="toc-backref" href="#id10">Accessing Processor Context</a><a class="headerlink" href="#accessing-processor-context" title="Permalink to this headline"></a></h2>
-            <p>As we have mentioned in <href=#defining-a-stream-processor>Defining a Stream Processor<\href>, a <code>ProcessorContext</code> control the processing workflow, such as scheduling a punctuation function, and committing the current processed state.</p>
+            <p>As we have mentioned in the <a href=#defining-a-stream-processor>Defining a Stream Processor</a> section, a <code>ProcessorContext</code> control the processing workflow, such as scheduling a punctuation function, and committing the current processed state.</p>
```
Comment (guozhangwang) [inline 20/streams/developer-guide/processor-api.html:357]: :)

**Review by guozhangwang — APPROVED** [review]
Thanks @rajinisivaram !

**Comment by rajinisivaram** [comment]
@guozhangwang Thanks for the review, merging to asf-site.

**Pull request merged**